goodtables/pipeline/pipeline.py

```diff
@@ -144,6 +145,16 @@ def get_pipeline(self):
 
         return pipeline
 
+    def validate_processors(self):
+        """Validates the processors parameter"""
+
+        if not isinstance(self.processors, (list, tuple, set)):
+            msg = 'The \'processors\' argument must be a list, tuple or set.'
+            raise exceptions.InvalidPipelineOptions(msg)
+
+        for processor_name in self.processors:
```

Comment:
I chose to iterate though all the processors and let the `resolve_processor` raise the error if a processor in invalid. If you have a better idea of how to handle this, please share.